vars/folioHelm.groovy

```diff
@@ -167,76 +167,38 @@ String generateModuleValues(RancherNamespace ns, String moduleName, String modul
-//    } else {
-//        repository = moduleVersion.contains('SNAPSHOT') ? "folioci" : "folioorg"
-//    }
+
   moduleConfig << [image         : [repository: "${repository}/${moduleName}",
                                     tag       : moduleVersion],
                    podAnnotations: [creationTimestamp: "\"${LocalDateTime.now().withNano(0).toString()}\""]]
 
-// Enable R/W split
-//  if (ns.enableRwSplit && Constants.READ_WRITE_MODULES.contains(moduleName)) {
-//    moduleConfig << [readWriteSplitEnabled: "true"]
-//  }
+/**
+ * Modules feature switcher
+ */
 
-//    // Enable JMX metrics
+// TODO Enable JMX metrics once prometheus will work
 //    if (Constants.JMX_METRICS_AVAILABLE[moduleName]) {
 //        def action = compare.compareVersion(Constants.JMX_METRICS_AVAILABLE[moduleName], moduleVersion)
 //        if (action == "upgrade" || action == "equal") {
 //            moduleConfig['javaOptions'] += " -javaagent:./jmx_exporter/jmx_prometheus_javaagent-0.17.2.jar=9991:./jmx_exporter/prometheus-jmx-config.yaml"
 //        }
 //    }
 
-//Enable extra env
-//  if (Constants.CONSORTIUM_ENABLED.contains(moduleName) && ns.enableConsortia) {
-//    moduleConfig << [consortiumEnabled: "true"]
-//  }
-//
-//  //Enable cross tenant extra env
-//  if (moduleName == 'mod-authtoken' && ns.enableConsortia) {
-//    moduleConfig['javaOptions'] += ' -Dallow.cross.tenant.requests=true'
-//  }
-//  //Enable RTR functionality with env value
-//  if (params.RTR) {
-//    moduleConfig << [rtrEnabled: "true"]
-//  }
-
-//  if (Constants.CONSORTIUM_ENABLED.contains(moduleName) && ns.enableConsortia) {
-//    moduleConfig << [consortiumEnabled: "true"]
-//  }
-//
-//  //Override default mdi-slicing, in case of minio
-//  if (params.S3_BUCKET == 'built-in' && moduleName == 'mod-data-import') {
-//    moduleConfig << [disEnabled: false]
-//    println("mod-data-import slicing was not requested...!")
-//  }
-//
-//  //Enable cross tenant extra env as default option
-//  if (moduleName == 'mod-authtoken') {
-//    moduleConfig['javaOptions'] += ' -Dallow.cross.tenant.requests=true'
-//  }
-//
-//  //Enable cross tenant extra env as default option
-//  if (moduleName == 'mod-bulk-operations' && params.NAMESPACE == 'sprint') {
-//    moduleConfig['javaOptions'] += ' -Dspring.servlet.multipart.max-file-size=40MB'
-//    moduleConfig['javaOptions'] += ' -Dspring.servlet.multipart.max-request-size=40MB'
-//  }
-//
-//  //Enable RTR functionality with env value
-//  if (params.RTR) {
-//    moduleConfig << [rtrEnabled: "true"]
-//  }
-//
-//  if (ns.getClusterName() == 'folio-dev') {
-//    moduleConfig << [modSearchDev: "true"]
-//    moduleConfig << [modInventoryStorageDev: "true"]
-//    moduleConfig << [modEntitiesLinksDev: "true"]
-//  }
-//
+  //Allow (in consortia setups) or deny cross tenant requests
+  if (moduleName == 'mod-authtoken' && ns.enableConsortia) {
+    moduleConfig['extraJavaOpts'] += ['-Dallow.cross.tenant.requests=true']
```

Comment:
We have agreed to have cross tenants requested enabled by default...

Reply:
resolved